src/SFA.DAS.AANHub.Database/Tables/Employer.sql

```diff
@@ -2,10 +2,10 @@
 (
     [MemberId] UNIQUEIDENTIFIER NOT NULL PRIMARY KEY, 
     [AccountId] BIGINT NOT NULL, 
-    [UserId] BIGINT NULL, 
-    [Organisation] NVARCHAR(250) NULL, 
-    [Email] NVARCHAR(256) NULL, 
-    [Name] NVARCHAR(200) NULL, 
-    [LastUpdated] DATETIME NULL
+    [UserRef] UNIQUEIDENTIFIER NOT NULL, 
+    [Organisation] NVARCHAR(250) NOT NULL, 
+    [Email] NVARCHAR(256) NOT NULL, 
+    [Name] NVARCHAR(200) NOT NULL, 
+    [LastUpdated] DATETIME NOT NULL
     CONSTRAINT [FK_Employer_Member] FOREIGN KEY ([MemberId]) REFERENCES [Member]([Id])
```

Comment:
Need to add a unique constraint `CONSTRAINT [UK_Employer] UNIQUE ([UserRef])`

Reply:
Added